test/grpc/integration/codec_test.exs

```diff
@@ -20,32 +20,35 @@ defmodule GRPC.Integration.CodecTest do
   defmodule HelloServer do
     use GRPC.Server,
       service: Helloworld.Greeter.Service,
-      codecs: [GRPC.Codec.Proto, GRPC.Codec.Erlpack]
+      codecs: [GRPC.Codec.Proto, GRPC.Codec.Erlpack, GRPC.Codec.WebText]
```

Comment:
I think some tests need to be added here to ensure this does indeed make a difference (it puts the proper headers, etc)